### PR TITLE
changed calculations to use big.js

### LINKED ISSUE #27: Use Big.js for price calculations
Because JavaScript, we need to use big.js to calculate numbers for price/shares/amount, etc. Otherwise we get the floating point bla problem.

### PR DESCRIPTION
**Pull request by codeofsumit**
Fixes annoying fees or prices like `2,960000000000008`.
Fixes #27 

Also changes Trade Republic calc & tests.
We should not round numbers in the import but only during presentations so calculations are as accurate as possible.

### CODE REVIEW

File: src/brokers/comdirect.js [hunk 1]
```diff
@@ -72,11 +73,11 @@ const findFee = textArr => {
     textArr[textArr.findIndex(t => t.includes('Zu Ihren')) + 1];
   const totalCost = totalCostLine.split('EUR').pop().trim();
 
-  return Math.abs(parseGermanNum(totalCost) - amount);
+  const diff = +(Big(parseGermanNum(totalCost)).minus(Big(amount)));
+  return Math.abs(diff);
```
Comment (carstenhag) [inline src/brokers/comdirect.js:77]: Big.abs() can be used as well

Comment (carstenhag) [reply src/brokers/comdirect.js:77]: What is the `+()` for? Tried to google it up, but didnt find anything :/ -- I guess it's for differenciating?

Comment (codeofsumit) [reply src/brokers/comdirect.js:77]: `+()` turns a `Big` into a JavaScript `number` again.
See https://mikemcl.github.io/big.js/
Under "How can I convert a Big number to a primitive JavaScript number?"

Comment (carstenhag) [reply src/brokers/comdirect.js:77]: Ah, I see, thanks.

**Pull request merged**